app/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,7 @@
     <string name="permission_needed_zooper_desc"><![CDATA[ <b>%1$s</b> needs permission to access your phone\'s storage in order to save Zooper assets!]]></string>
     <string name="download_cancelled">Download cancelled!</string>
     <string name="cmte_unavailable">CyanogenMod Theme Engine is not available to this device.</string>
+    <string name="suma_unavailable">Suma Launcher Engine is not available to this device.</string>
```

Comment:
i just use it for testing purpose. i forget to remove this string. Suma launcher work on every device. Suma Launcher is based on Launcher 3. which is supported by every device. please consider it. thanks